source/openapi-admin-v3.yaml

```
@@ -7740,7 +7743,7 @@ components:
             - date
             - timestamp
             - bool
-            - null
+            - 'null'
```

```suggestion
            - "null"
```
double-quotes for consistency (see existing [line 6155](https://github.com/mongodb/docs-app-services/blob/7507f4d2c7ed51ed31ceee08fc00826695e52786/source/openapi-admin-v3.yaml#L6155))